metrics-operator/controllers/common/providers/prometheus/common.go

```diff
@@ -0,0 +1,57 @@
+package prometheus
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	metricsapi "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1alpha3"
+	promapi "github.com/prometheus/client_golang/api"
+	"github.com/prometheus/common/config"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const userName = "user"
+const password = "password"
```

Comment:
let's rename those to make it clear they are keys for accessing their values in the secret, rather than actual username and password values

```suggestion
const secretKeyUserName = "user"
const secretKeyPassword = "password"
```